### PR TITLE
Insist about authentication in concepts documentation (ref #976)

### PR DESCRIPTION
**Pull request by leplatrem**
Ref #976 

A lot of work still has to be done regarding authentication when #795 will land.

### CODE REVIEW

**Comment by leplatrem** [comment]
Does not harm, merging.

File: docs/api/1.x/authentication.rst [hunk 1]
```diff
@@ -111,6 +111,9 @@ For each token, Kinto will calculate a unique user ID which is
 related to your Kinto instance. It uses a bit of cryptography and the value of
 the ``user_hmac_secret`` setting.
 
+In other words, every string provided in Basic Auth is valid, and leads to a
```
Comment (almet) [inline docs/api/1.x/authentication.rst]: Should this be "as Basic Auth"?

File: docs/concepts.rst [hunk 2]
```diff
@@ -65,6 +65,16 @@ Permissions can refer to the *group* URI instead of an individual user - this ma
...
 
+.. _concepts-authentication:
+
+Authentication
+==============
+
+Kinto doesn't provide users management, like user sign-up, password modification, etc.
+
+Kinto can rely on a third-party called "`Identity provider <https://en.wikipedia.org/wiki/Identity_provider>`_"
+to authenticate the incoming request and assign a :term:`user id`.
```
Comment (almet) [inline docs/concepts.rst]: Adding examples of third parties could be useful here in order to understand better.

**Comment by leplatrem** [comment]
THanks! I just pushed your suggestions to master ;)

**Pull request merged**